src/SonarQube.Client.Tests/Requests/DefaultConfiguration_Configure_Tests.cs

```diff
@@ -20,6 +20,7 @@
 
 using System;
 using System.Diagnostics;
+using System.Linq;
```

Comment:
This doesn't seem intentional
```suggestion
```

Reply:
Yep, It was added during one of the iterations of writing the tests and is not needed anymore